src/style.css

```
@@ -1 +1,44 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  padding: 1%;
+}
+
+.logo {
+  margin-left: 2.5%;
+  flex-grow: 3;
+}
+
+.nav__list {
+  display: inline-flex;
+  justify-content: space-evenly;
+  list-style: none;
+}
```

```suggestion
.nav__list {
  display: flex;
  list-style: none;
}
}
```
this will fix the alignment

---

Also works